deps/spidershim/test/value.cc

```diff
@@ -1434,3 +1434,177 @@ TEST(SpiderShim, ArrayBuffer_NeuteringApi) {
   Local<Float64Array> f64a =
       CreateAndCheck<Float64Array, 8>(buffer, 8, 127);
 }
+
+static inline v8::Local<v8::Script> v8_compile(v8::Local<v8::String> x) {
+  v8::Local<v8::Script> result;
+  if (v8::Script::Compile(v8::Isolate::GetCurrent()->GetCurrentContext(), x)
+          .ToLocal(&result)) {
+    return result;
+  }
+  return v8::Local<v8::Script>();
+}
+
+static inline v8::Local<v8::Value> CompileRun(v8::Local<v8::String> source) {
```

Comment:
I don't think it will make anything harder, it will make your test consistent with the existing ones.  I'll do this myself, no point debating this.  :-)